pandas/tests/generic/test_generic.py

```diff
@@ -897,3 +897,13 @@ def test_flags_identity(self, as_frame):
         assert s.flags is s.flags
         s2 = s.copy()
         assert s2.flags is not s.flags
+
+    def test_slice_incomplete_datetimes(self):
+        # GH36953 and GH35509
+        increment = pd.Series(
```

Comment:
I know this is the OP but is it possible to explicitly construct it? without using the `cumsum`

Reply:
done